### PR TITLE
MolNet BBBC Load Function fix

### LINKED ISSUE #3675: Labels as an array of zeros when trying to load BBBC001 from MolNet
## 🐛 Bug

When loading the labels for image set BBBC001 from MolNet, labels appear as a numpy array of zeros
Same goes for BBBC002.

## To Reproduce

import deepchem as dc
from deepchem.feat.molecule_featurizers import MolGraphConvFeaturizer
import numpy as np
import matplotlib.pyplot as plt
import pandas as pd

dataset_dc = dc.molnet.load_bbbc001()
tasks, dataset, transformers = dataset_dc
train, valid, test = dataset

x,y,w,ids = train.X, train.y, train.w, train.ids

print(y)

## Current output

[0. , 0. , 0. , 0.]

## Expected behavior

Get output as an array of the mean values of 2 manual counts of cells in each sample's labels .txt file

## Environment

* OS: Colab
* Python version: 3.10.12
* DeepChem version: 2.7.1
* RDKit version (optional):
* TensorFlow version (optional):
* PyTorch version (optional):
* Any other relevant information:

### PR DESCRIPTION
**Pull request by aaronrockmenezes**
## Description

Fixed #3675

## Type of change

Please check the option that is related to your PR.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - In this case, we recommend to discuss your modification on GitHub issues before creating the PR
- [ ] Documentations (modification for documents)

## Checklist

- [x] My code follows [the style guidelines of this project](https://deepchem.readthedocs.io/en/latest/development_guide/coding.html)
  - [x] Run `yapf -i <modified file>` and check no errors (**yapf version must be  0.32.0**)
  - [x] Run `mypy -p deepchem` and check no errors
  - [x] Run `flake8 <modified file> --count` and check no errors
  - [x] Run `python -m doctest <modified file>` and check no errors
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: deepchem/molnet/load_function/bbbc_datasets.py [hunk 1]
```diff
@@ -29,8 +31,15 @@ def create_dataset(self) -> Dataset:
         if not os.path.exists(labels_file):
             dc.utils.data_utils.download_url(url=BBBC1_LABEL_URL,
                                              dest_dir=self.data_dir)
+        labels_table = pd.read_csv(labels_file, delimiter="\t")
```
Comment (aaronrockmenezes) [inline deepchem/molnet/load_function/bbbc_datasets.py:34]: The documents for this say that the true labels are the averages of the cells counts done by the 2 professional human counters.

**Review by rbharath — COMMENTED** [review]
LGTM

**Review by rbharath — APPROVED** [review]
LGTM
